cmd/autoscaler/main.go

```diff
@@ -179,24 +179,26 @@ func scalerConfig(logger *zap.SugaredLogger) *autoscaler.DynamicConfig {
 	return dynConfig
 }
 
+var requiredLabels = []string{serving.RevisionLabelKey, serving.ServiceLabelKey, serving.ConfigurationLabelKey}
```

Comment:
Generally i'm not a van of mutable globals and would prefer to make this at runtime (slight computational cost but less risk)

If we keep this though one small nit is we can make the array size constant with [...]string

Reply:
I made it `[3]string` since it's not going to change most likely.
Unfortunately go does not support the notion of immutable slices, but I don't want to recompute obviously constant list each time (since this function is used for each revision basically).